coalib/parsing/FilterHelper.py

```diff
@@ -20,11 +20,8 @@ def apply_filter(filter_name, filter_args, all_bears=None):
             get_all_bears)
         all_bears = get_all_bears()
     if not is_valid_filter(filter_name):
-        raise InvalidFilterException('{!r} is an invalid filter. '
-                                     'Available filters: {}'.format(
-                                         filter_name,
-                                         ', '.join(sorted(
-                                             available_filters))))
+        raise InvalidFilterException(
+            filter_name, available_filters)
```

Comment:
As mentioned in https://github.com/coala/coala/pull/4821/files/f47beedaade7a74a7cf71a7a31283522006dcc43#r150385738 ... Are there ever other `available_filters` than `FilterHelper.available_filters`? IMO the parameter is unnecessary

Reply:
That commit range is not found. Which commit are you talking about?

Reply:
He's asking to remove the 2nd param. Its unnecessary. Directly use it from `FilterHelper` in the exception constructor.

Reply:
@userzimmermann actually we can't remove `available_filters`, otherwise, there will be recursive import errors.